src/typeDefs/schema.graphql

```diff
@@ -152,6 +152,18 @@ input SubmissionSearchFilters {
   isUnderReview: Boolean
   isApproved: Boolean
   isRejected: Boolean
+  orderBy: [OrderBy!]
+  limit: Int
```

Comment:
this should be marked as optional

Reply:
this looks like it went the opposite direction 😆 
let's make `orderBy` and `limit` both optional.